modules/virt-adjust-CPU-VM-hotplug-instancetype.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// * /virt/managing_vms/advanced_vm_management/virt-NUMA-topology.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-adjust-CPU-VM-hotplug-instancetype_{context}"]
+= Adjusting or disabling the CPU hot plug by instance type
```

Comment:
Why adjusting or disabling? The use case is just to disable vCPU hot plug right?